src/components/FloatingDropdown/FloatingDropdown.e2e.ts

```diff
@@ -0,0 +1,26 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { floatingDropdownTestkitFactory } from '../../testkit/protractor';
+
+describe('floatingDropdown', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Components',
+    story: 'FloatingDropdown',
+    withExamples: true,
+  });
+  const dataHook = 'storybook-FloatingDropdown';
+
+  beforeEach(() => browser.get(storyUrl));
+
+  it('should render', async () => {
```

Comment:
If we don't need any other tests in this file, then we can remove the file altogether.
Our e2e tests are used when we need a DOM for testing certain stuff.
In this case, we might need it to test that the dropdown opens as expected, etc.
However, we might also achieve this with visual tests.